README.md

```diff
@@ -209,3 +209,90 @@ Execute installation script
       *  [Vault](https://github.com/challabeehyv/sunbird-devops/blob/main/deploy-as-code/helm/v2/README.md#vault-deployment)
    * [Esignet](https://github.com/mosip/esignet/tree/v1.4.1/helm)
    * [Certify](https://github.com/mosip/inji-certify/tree/v0.9.1/helm/inji-certify)
+
+
+## Configuration docs for Postgres Plugin
+
+- Pre-requisites
+     Authorisation Provider should expose the unique identifier in the `sub` field of the JWT token.
+       Eg. If one is using eSignet with mock-identity-system:0.10.0 and above it can be achieved by setting:
+       ```mosip.mock.ida.kyc.psut.field=individualId```
+       where individualId will be the identifier to locate the identity in the expected identity registry.
+
+1. Create the tables with all the fields that should be returned by the Postgres Data Provider Plugin within the certify postgres database.
+    - Refer the following query for insertion in DB:
+      ```
+        CREATE TABLE certify.<table_name> (
```

Comment:
are we finalizing this design for certify pg plugin?
data present within certify DB?

Reply:
@vharsh  @Piyush7034

Reply:
This doc refers to the configuration setup for postgres plugin as requested by Likhitha. So it was suggested to add this in the certify docs for reference. This should be updated based on future changes.

Reply:
No, but this is being given as a handover to the QA team and a minimal documentation is required for the same, can we drop this plugin from the scope?

Reply:
This document is now moved to digital-credential-plugins repo under postgres-dataprovider-plugin project as suggested by @vishwa-vyom